wiki/Gameplay/Fingerlock/de.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - wrist lock
+  - wristlock
+  - wrist-lock
+  - finger-lock
+  - finger lock
+---
+
+# Fingerlock
+
+*Nicht zu verwechseln mit [Notelock](/wiki/Gameplay/Notelock) oder [Mind block](/wiki/Gameplay/Mind_block).*
+
+**Fingerlock** (auch `Finger-Lock` oder `Finger Lock` genannt) ist ein Begriff, der das Gefühl beschreibt, die Kontrolle über die Finger zu verlieren, während man osu! spielt. Obwohl der Begriff nicht spezifisch für osu! ist, wird er von Spielern häufig verwendet, um ihre Frustration über Situationen auszudrücken, in denen ein Spieler weiß, wann und wo er ein [Hit-Objekt](/wiki/Hit_object) treffen muss, aber unerklärlicherweise die Taste einfach nicht drückt, was zu einem Gefühl führt, als sei der Finger "gesperrt" (engl. "locked"). Daher auch der Name "Fingerlock".
+
+"Fingerlock" kann gelegentlich auch als `wrist lock`, `wristlock` oder `wrist-lock` bezeichnet werden.
+
+## Ursachen
+
+*Vorsicht: Fingerlock ist **kein** Krankheitszustand. Alle angeblichen Forschungsergebnisse oder allgemein bekannten Informationen der osu! Community über die Ursache und/oder die langfristigen Auswirkungen sind **nicht als wissenschaftlich** zu betrachten. Die osu!-Community ist nicht qualifiziert, medizinische Ratschläge zu erteilen; verwende das osu!-Wiki nicht, um medizinische Zustände zu diagnostizieren.*
```

Comment:
```suggestion
*Vorsicht: Fingerlock ist **kein** Krankheitszustand. Alle angeblichen Forschungsergebnisse oder allgemein bekannten Informationen der osu! Community über die Ursache und/oder die langfristigen Auswirkungen sind **nicht als wissenschaftlich** zu betrachten. Die osu!-Community ist nicht qualifiziert, medizinische Ratschläge zu erteilen; verwende das osu!-Wiki nicht, um medizinische Gegebenheiten zu diagnostizieren.*
```

Reply:
Both is fine.

Reply:
what do you think about `medizinische Beschwerden`

Reply:
`medizinische Beschwerden` sounds even better 👍